Dockerfile

```diff
@@ -10,4 +10,7 @@ ADD dist/docker_hostdns-${HOSTDNS_VERSION}-py3-none-any.whl /usr/local/share/
 RUN pip install /usr/local/share/docker_hostdns-${HOSTDNS_VERSION}-py3-none-any.whl \
     && rm -rf /root/.cache
 
+VOLUME ["/run/docker.sock"]
```

Comment:
Not sure about it as it is not a data point where container creates data and if someone will not mount socket I'm assuming it will create directory in container, so not really as expected.

Can you remove this line? Otherwise this PR looks good :)